src/orion/benchmark/assessment/averagerank.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+:mod:`orion.benchmark.assessment` -- Average Rank Assessment
+================================================================
+
+.. module:: assessment
+   :platform: Unix
+   :synopsis: Benchmark algorithms with average rank.
+
+"""
+
+from collections import defaultdict
+
+import numpy as np
+import pandas as pd
+import plotly.express as px
+
+from orion.benchmark.base import BaseAssess
+
+
+class AverageRank(BaseAssess):
+    """
+    For each algorithm, run fixed number of Experiment, average the rank of trials for
+    the same algorithm at the same trial sequence order.
+    For the performance of trials in an Experiment, instead using the actual trial objective value,
+    here we use the best objective value in the same Experiment until the particular trial.
+    """
+
+    def __init__(self, task_num=1):
+        super(AverageRank, self).__init__(task_num=task_num)
+
+    def plot_figures(self, task, experiments):
+        """
+        Generate a `plotly.graph_objects.Figure`
```

Comment:
You should explain here what the plots are. Regret curves?

Reply:
Not sure if it will have a known name like `Regret curve`.  My idea is that the assessment (or purpose) will be the plot name.

Reply:
Ok, but we should explain in a short (or couple of short) sentence(s) what the plot is a bout. A bit like we would describe a figure in a paper.